src/components/table-toolbar/KeycloakDataTable.tsx

```diff
@@ -215,24 +226,14 @@ export function KeycloakDataTable<T>({
             selected: !!selected.find(
               (v) => _.get(v, "id") === _.get(value, "id")
             ),
-            isOpen: false,
-            cells: columns.map((col) => {
-              if (col.cellRenderer) {
-                return { title: col.cellRenderer(value) };
-              }
-              return _.get(value, col.name);
-            }),
+            isOpen: isDetailColumnsEnabled(value) ? false : undefined,
```

Comment:
Is the result of this expression not always `false` or `undefined`? Perhaps you can default to `false` here?
```suggestion
            isOpen: false,
```

Reply:
false is not the same as undefined when we always set it to false it will always show the collapse icon we only want that when detail columns are enabled

Reply:
I see, thanks for clarifying.